kebab/src/websocket/raw.ts

```diff
@@ -1,26 +1,26 @@
 import WebSocket from "isomorphic-ws";
 import ReconnectingWebSocket from "reconnecting-websocket";
 import { v4 as generateUuid } from "uuid";
-import { User, UUID } from "./entities";
+import { User, UUID } from "..";
 
 const heartbeatInterval = 8000;
 const apiUrl = "wss://api.dogehouse.tv/socket";
 // const apiUrl = "ws://localhost:4001/socket";
 const connectionTimeout = 15000;
 
 export type Token = string;
-export type FetchID = UUID;
+export type Ref = UUID;
 export type Opcode = string;
 export type Logger = (
   direction: "in" | "out",
   opcode: Opcode,
   data?: unknown,
-  fetchId?: FetchID,
```

Comment:
you need to keep fetchId until all the new casts/calls have been put in, only the ones with keys like `room:ban`  work with ref

Reply:
oh